### PR TITLE
[QUEST] [A] 12499 and [H] 12500 - Return To Angrathar (lack of phasing after completing quest)

### PR DESCRIPTION
**Pull request by xml21**
**Changes proposed:**

-  Fix of phasing after completing Return To Angrathar quest for Horde & Alliance.

Before fix nothing happened right after quest completion, Player needed to go to 4171 area (Right at the wrath gate) to see the phase changes (fire & dragons). NPC was still visible after Wrathgate movie.

WowHead links:
[H] http://www.wowhead.com/quest=12500/return-to-angrathar
[A] http://www.wowhead.com/quest=12499/return-to-angrathar

**Issues addressed:** 

None

**Tests performed:** (Does it build, tested in-game, etc)

Tested

### CODE REVIEW

**Comment by ShinDarth** [comment]
Thanks for your fix, but that is not the correct way to add it.

Here you can find the detailed instructions:

https://github.com/azerothcore/azerothcore-wotlk/wiki/Dealing-with-SQL-files

**Comment by xml21** [comment]
Done a commit with Update wrapper

File: data/sql/base/db_world/spell_area.sql [hunk 1]
```diff
@@ -526,9 +526,9 @@ INSERT INTO `spell_area` VALUES
 (58730,4585,0,0,0,0,2,1,64,11),
 (58730,4589,0,0,0,0,2,1,64,11),
 (58730,4612,0,0,0,0,2,1,64,11),
-(58932,4171,12499,0,0,0,2,1,64,11),
+(58932,4169,12499,0,0,0,2,1,64,11),
 (58932,4171,12499,0,0,1101,2,1,64,11),
-(58932,4171,12500,0,0,0,2,1,64,11),
+(58932,4170,12500,0,0,0,2,1,64,11),
```
Comment (ShinDarth) [inline data/sql/base/db_world/spell_area.sql]: you should revert this file diff

File: data/sql/updates/pending_db_world/rev_1482010582878343200.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+UPDATE `spell_area` SET `area`='4169' WHERE (`spell`='58932') AND (`area`='4171') AND (`quest_start`='12499') AND (`aura_spell`='0') AND (`racemask`='0') AND (`gender`='2') LIMIT 1;
+
+UPDATE `spell_area` SET `area`='4170' WHERE (`spell`='58932') AND (`area`='4171') AND (`quest_start`='12500') AND (`aura_spell`='0') AND (`racemask`='0') AND (`gender`='2') LIMIT 1;
```
Comment (ShinDarth) [inline data/sql/updates/pending_db_world/rev_1482010582878343200.sql]: please add a new line at the end of the file

Comment (xml21) [inline data/sql/updates/pending_db_world/rev_1482010582878343200.sql]: done

Comment (ShinDarth) [inline data/sql/updates/pending_db_world/rev_1482010582878343200.sql]: 👍

**Comment by Irome** [comment]
I'm testing this quest. but I can not figure out what kind of quest is. what must I do, to be able to take this quest?

I tried it with a [A] paladin level 80, I went to (Alexstrasza the Life-Binder) to take the quest (Return To Angrathar), but the NPC did not have any quests to give me. I tried to change the reputation, but nothing has changed.

I tried to add the quest with the controls, but the quest is not active, and nothing happens.

Is this quest for a specific class?

**Comment by xml21** [comment]
- '.quest add 12500 for Horde side'
- '.quest add 12499 for Alliance'

Its very long chain for all dragonblight quests

**Comment by Irome** [comment]
I tested it, and everything works as it should. xml21 great job.

**Pull request merged**